pkg/controller/storagecluster/update.go

```diff
@@ -86,6 +87,16 @@ func (c *Controller) rollingUpdate(cluster *corev1.StorageCluster, hash string)
 		return err
 	}
 
+	// If px version is greater than 2.15 then delete px-api pods along with px pods
+	pxVersion := util.GetPortworxVersion(cluster)
+	supportedPxVersion, _ := version.NewVersion("2.15")
```

Comment:
The supported version should be 2.13 and not 2.15. Also, please extract a constant for it.